src/YCP_UI.h

```diff
@@ -53,6 +53,7 @@ class YCP_UI
     static YCPValue	AskForExistingDirectory		( const YCPString & startDir, const YCPString & headline );
     static YCPValue 	AskForExistingFile		( const YCPString & startDir, const YCPString & filter, const YCPString & headline );
     static YCPValue 	AskForSaveFileName		( const YCPString & startDir, const YCPString & filter, const YCPString & headline );
+    static void         AskForWidgetStyle               ();
```

Comment:
NP: different indentation?

Reply:
Probably tabs vs. blanks. Diffs, in particular in GitHub, don't work well with that.